phpunit.xml

```diff
@@ -1,25 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit backupGlobals="false"
-	bootstrap="test/bootstrap.php"
-	backupStaticAttributes="false"
-	colors="true"
-	convertErrorsToExceptions="true"
-	convertNoticesToExceptions="true"
-	convertWarningsToExceptions="true"
-	processIsolation="false"
-	stopOnFailure="false"
-	syntaxCheck="false"
+    backupStaticAttributes="false"
+    bootstrap="Tests/bootstrap.php"
```

Comment:
Change this to point phpunit straight at the composer `autoload.php` file. One less file that serves no purpose other than to include another file.